airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -14,7 +14,7 @@ acceptance_tests:
     tests:
       - config_path: "secrets/config.json"
         backward_compatibility_tests_config:
-          disable_for_version: "3.3.0"
+          disable_for_version: "3.6.0" # new streams added: Cards, Payment Methods
```

Comment:
Please clarify in the comment that there is no actual breaking change and that it's only because the schema files changed

Reply:
updated